### DISCUSSION TITLE
Efficient way to use Isar in all branch widget without passing one instant?

### DISCUSSION THREAD

**Discussion by anandsubbu007** [isar/isar#125] [Help] [answered]
I went through isar documentation.
it is really awesome. thanks for your support to us.

I would like to know an efficient way to use isar in a larger project.
in your sample project isar instance is passed between widgets.

how do I save that instance and use it whenever I want. 
in hive, I open the DB and save all data in class and access through provider.

I don't know how to implement this in isar.

any suggestion?

**Comment by simc** [comment, marked as answer]
I would put the instance in a provider or use any dependency injection method :)
